core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -562,6 +570,7 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
         |
         | Spark on YARN only:
         |  --queue QUEUE_NAME          The YARN queue to submit to (Default: "default").
+        |  --priority PRIORITY         The priority of your YARN application.
```

Comment:
Ur, I'm a little negative for this because this can be used by other resource manager.

Why don't we use `--conf spark.yarn.priority` simply?

Reply:
Yeah, same question.

Reply:
> Why don't we use --conf spark.yarn.priority simply?

You are right, we can use `-- conf spark.yarn.priority `, but `-- priority` is more simple and easy to remember by users. similar to `--queue`.

Reply:
Wouldn't this extend to lots of other arguments? I think the ones that exist are very common and widely applicable. There are probably a few too many already.

Reply:
> I think the ones that exist are very common and widely applicable.
@srowen  Got it ~ I will close this pr to end this unnecessary arguments, although it is widely used and accepted by our users ：）